cmd/cdi-controller/controller.go

```diff
@@ -300,11 +301,11 @@ func addCrdInformerEventHandlers(crdInformer cache.SharedIndexInformer, extclien
 				startSmartController(extclient, mgr, log)
 			}
 		},
-	})
+	}, time.Duration(time.Minute))
 }
 
 func startSmartController(extclient extclientset.Interface, mgr manager.Manager, log logr.Logger) {
-	if controller.IsCsiCrdsDeployed(extclient) {
+	if controller.IsCsiCrdsDeployed(extclient, log) {
 		log.Info("CSI CRDs detected, starting smart clone controller")
 		if _, err := controller.NewSmartCloneController(mgr, log, installerLabels); err != nil {
```

Comment:
Have to make sure to not do this multiple times I think

Reply:
Yeah, this probably worked because we never tried to do it more than once.